src/build-tx/coinselect-lib/inputs/accumulative.js

```diff
@@ -25,32 +23,32 @@ export default function accumulative(utxos, outputs, feeRate, options) {
         if (utxoValue.isNaN()
             || utxoValue.comparedTo(new BigNumber(utxoFee)) < 0) {
             if (i === utxos.length - 1) {
+                const fee = utils.getFee(feeRateNumber, bytesAccum + utxoBytes, options, outputs);
                 return {
-                    fee: (feeRateNumber * (bytesAccum + utxoBytes)).toString(),
+                    fee: fee.toString(),
                 };
             }
         } else {
             bytesAccum += utxoBytes;
             inAccum = inAccum.plus(utxoValue);
             inputs.push(utxo);
 
-            const fee = feeRateNumber * bytesAccum;
+            const fee = utils.getFee(feeRateNumber, bytesAccum, options, outputs);
             const outAccumWithFee = outAccum.isNaN()
-                ? new BigNumber(0) : outAccum.plus(new BigNumber(fee));
+                ? new BigNumber(0) : outAccum.plus(fee);
```

Comment:
Not sure if needed or not but `fee` doesn't return a `BigNumber` (previously was wrapped in a `BigNumber constructor`).

Reply:
`outAccum` is a BigNumber `fee` is string

any operations on BigNumber returns BigNumber (in this case .plus), previous implementation have unnecessary conversion to BigNumber from string (string is accepted parameter of .plus)